### PR TITLE
Bug 1880018: Fix for Topology drag crash, update react-topology package

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**Fixes**
https://issues.redhat.com/browse/ODC-4834

**Description**
Update version of patternfly react-topology package to pickup a fix for crashing the UI when a node is being dragged and a data update occurs. Update topology list and graph components to use their own `Visualization`. This fixes an initial layout issue and prevents the list view from actually laying out nodes that are never drawn. Some minor adjustments to handle difference in layout call ordering done in the react-topology package.

**Browser conformance**: 
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

/kind bug

### CODE REVIEW

**Comment by christianvogt** [comment]
topology is loading in the top left corner now instead of centered

**Comment by jeff-phillips-18** [comment]
@christianvogt Updated and fixed the layout issue.

File: frontend/packages/dev-console/src/components/topology/TopologyView.tsx [hunk 1]
```diff
@@ -279,61 +236,42 @@ export const TopologyView: React.FC<ComponentProps> = ({
...
           namespace={namespace}
           application={application}
-          selectedIds={selectedIds}
           onSelect={onSelect}
+          setVisualization={onVisualizationChange}
         />
       ),
-    [application, namespace, selectedIds, showGraphView, visualization],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [application, filteredModel, namespace, showGraphView],
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/topology/TopologyView.tsx:259]: Why don't you include `onVisualizationChange` in the dependencies?

File: frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx [hunk 2]
```diff
@@ -37,26 +46,55 @@ interface TopologyListViewPropsFromDispatch {
...
+        visualization.fromModel(model);
+        const selectedItem = selectedId ? visualization.getElementById(selectedId) : null;
+        if (!selectedItem || !selectedItem.isVisible()) {
+          onSelect();
+        } else {
+          onSelect(selectedItem);
+        }
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [model, visualization]);
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx:97]: Missing `onSelect`

**Pull request merged**